src/storage.tsx

```diff
@@ -41,7 +41,7 @@ export function useStorageTask<T = unknown>(
   const observableId = `storage:task:${ref.toString()}`;
   const observable$ = _fromTask(task);
 
-  return useObservable(observableId, observable$, options ? options.initialData : undefined);
+  return useObservable(observableId, observable$, options ? options.initialData ?? options.startWithValue : undefined);
```

Comment:
Before and after the change this doesn't look correct to me, shouldn't it just pass options? `useObservable`'s third argument is an object.

Reply:
great catch, fixed